libraries/Microsoft.Bot.Builder.Azure/AzureBlobStorage.cs

```diff
@@ -117,36 +117,33 @@ private static string GetBlobName(string key)
                 {
                     var blobName = GetBlobName(key);
                     var blobReference = this.Container.Value.GetBlobReference(blobName);
-                    using (var memoryStream = new MemoryStream())
+                    var jsonSerializer = JsonSerializer.Create(SerializationSettings);

```

Comment:
Suggest not creating a new `JsonSerializer` every time. Configure an instance in a static field and reuse it.